### ISSUE TITLE
Possibility to log in using access token

### ISSUE THREAD

**Issue by iLexiconn** [RX14/jMCLaunchLib#13]

**Comment by RX14** [comment]
Is there any particular reason this is required? it should be easy enough just asking why is all.

As a workaround you can create the token cache file before running the `LaunchTaks` to provide the accesstoken.

Should the token be refreshed?

**Comment by RX14** [comment]
You could join esper#RX14 if you want to speak more

**Comment by RX14** [comment]
```
<iLexiconn> heyo
<RX14> hi
<iLexiconn> I need the option to support multiple accounts but still cache the accesstokens. I was thinking maybe set a custom MinecraftAuthResult in LaunchTask
<RX14> it does cache for multiple accounts
<iLexiconn> oh really? that's great
<RX14> the cache is already map from username to token
<RX14> :3
<RX14> https://github.com/RX14/jMCLaunchLib/blob/master/src/main/groovy/uk/co/rx14/jmclaunchlib/tasks/LoginTask.groovy#L53
<iLexiconn> ah, neat
<iLexiconn> Nevermind the issue then :P
<iLexiconn> Thanks
```

**Comment by iLexiconn** [comment]
I think I found an usage for this. Currently I log the user in before my launcher downloads the mods. After that's done, it has to login again. A method to do this could make it possible to reuse the `MinecraftAuthResult` from earlier.

**Comment by iLexiconn** [comment]
And with that, maybe I could also use a method to login with data from the auth cache. :stuck_out_tongue:

**Comment by RX14** [comment]
Yeah that sounds like a usecase... shouldn't be too hard

**Comment by RX14** [comment]
If I have added a simple method to get an always-valid `MinecraftAuthResult` by supplying only a username(usually email) and a `PasswordSupplier`, so you can use that to login, then the second launcher login will simply refresh the token. Will that do?

**Comment by iLexiconn** [comment]
Yup. :)

**Comment by RX14** [comment]
Released version `0.4.1` to jcenter :)
